release-notes-3.1.1.md

```diff
@@ -0,0 +1,56 @@
+# Release Notes
+
+While the 3.1.1 release makes no changes to requirements of the OpenAPI 3.1.0 specification, it does introduce a number of notable improvements, including:
+- Expands and clarifies a number of explanations, including several new appendices with supplementary details
+- Focuses on technical specifics by moving examples and additional documentation now published at [learn.openapis.org](https://learn.openapis.org)
+- Declares that the HTML specifications at [spec.openapis.org](https://spec.openapis.org) are now the authoritative versions (formerly it was the Markdown source on GitHub)
+
+OpenAPI Description writers should mark their OpenAPI Descriptions with the version of the OpenAPI specification they used to write their specification, updating where possible.
```

Comment:
@earth2marsh commented that we needed to expand on this, but his remarks were lost in some of the reformatting. I'm not sure this belongs here in the release notes, should probably be in the announcement post instead.

Reply:
@lornajane agree on any expansion being more suitable for the announcement post.